### ISSUE TITLE
CDATA Sections

### ISSUE THREAD

**Issue by borodiliz** [locize/xliff#30]
## 🐛 Bug Report

Thanks for this project! 

I'm using XILF files generated with Symfony translator, lot of them, includes CDATA Sections as [specified here](https://docs.oasis-open.org/xliff/v1.2/xliff-profile-html/xliff-profile-html-1.2-cd02.html#General_CDATA). Is CDATA supported?

Not working (empty string given):
```xml
      <trans-unit id="ALVJf0m" resname="general_validateSubmit">
        <source>general_validateSubmit</source>
        <target><![CDATA[Validate & Submit]]></target>
      </trans-unit>
```

Working:

```xml
      <trans-unit id="ALVJf0m" resname="general_validateSubmit">
        <source>general_validateSubmit</source>
        <target>Validate &amp; Submit</target>
      </trans-unit>
```
## To Reproduce

Based on [this](https://netbasal.gitbook.io/transloco/recipes/xliff-loader) 

```js

  getTranslation(langPath: string) {
    return this.http
      .get(`${TranslocoHttpLoader.getAssetsPath()}/i18n/myfile.${langPath}.xlf`, {
        responseType: 'text' as 'text',
      })
      .pipe(mergeMap((translation) => xliff(translation).then((toJSON) => toTranslationFormat(toJSON))));
  }
}

function toTranslationFormat(json): Translation {
  const obj = json.resources['file.ext'];
  return Object.keys(obj).reduce((acc, key) => {
    const { source } = obj[key];
    acc[source] = obj[key].target; //EMPTY STRING GIVEN HERE FOR CDATA TRANSLATIONS
    return acc;
  }, {});
}
```

## Expected behavior

Non empty strings for CDATA translations

## Your Environment

xliff 5.1.0 and:

```
Angular CLI: 10.0.2
Node: 12.16.3
OS: darwin x64

Angular: 10.0.3
... animations, common, compiler, compiler-cli, core, elements
... forms, language-service, platform-browser
... platform-browser-dynamic, router
Ivy Workspace: <error>

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.901.11
@angular-devkit/build-angular     0.1000.2
@angular-devkit/build-optimizer   0.901.11
@angular-devkit/build-webpack     0.901.11
@angular-devkit/core              9.1.11
@angular-devkit/schematics        10.0.2
@angular/cli                      10.0.2
@angular/google-maps              10.0.1
@ngtools/webpack                  9.1.11
@schematics/angular               10.0.2
@schematics/update                0.1000.2
rxjs                              6.6.0
typescript                        3.9.6
webpack                           4.42.0
```

**Thanks in advance!**

**Comment by adrai** [comment]
Currently, afk... didn’t test...
Can you create a test first, and if possible/needed try to create PR to fix it?

**Comment by adrai** [comment]
@probertson do you have any more insights, on why cdata values gets set to empty string?

**Comment by adrai** [comment]
Thank you!
Released in v5.2.0
